### PR TITLE
chore(ui): move deprecated components to deprecated_js

### PR DESCRIPTION
**Pull request by guoda-puidokaite**
# Summary

In this PR, deprecated components have been moved to the `deprecated_js` folder, fixing all imports, exports, and ensuring deprecated labels have been added in `Storybook`.

# Changes Made

- Moved deprecated files to `deprecated_js` (list below)
- Made sure all imports and exports are correct
- Made sure all deprecated components are marked as `deprecated` in Storybook

**Newly deprecated in Storybook with this PR**:

- [x] `AppIntro` (converted to TS)

**Previously deprecated in Storybook**:

- [x] `CheckboxRow` (converted to TS)
- [x] `ContentArea`
- [x] `ContentAreaWrapper`
- [x] `DataList` (https://github.com/cloudoperators/juno/issues/226):
  - [x] `DataList`
  - [x] `DataListCell`
  - [x] `DataListCheckboxCell`
  - [x] `DataListRow`
- [x] `Filter` https://github.com/cloudoperators/juno/issues/225: 
  - [x] `FilterInput`
  - [x] `FilterPill`
  - [x] `Filters`
- [x] `RadioRow` (converted to TS)
- [x] `SelectRow`
- [x] `SwitchRow`
- [x] `TextareaRow`
- [x] `TextInputRow`

# Related Issues

- Closes https://github.com/cloudoperators/juno/issues/662

# Testing Instructions

Storybook

# Checklist

- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have made corresponding changes to the documentation (if applicable).
- [x] My changes generate no new warnings or errors.

### CODE REVIEW

File: packages/ui-components/src/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 import "./global.scss"
 export { AppBody } from "./components/AppBody/AppBody.component"
-export { AppIntro } from "./components/AppIntro/AppIntro.component"
+export { AppIntro } from "./deprecated_js/AppIntro/AppIntro.component"
```
Comment (barsukov) [inline packages/ui-components/src/index.js:8]: Maybe we could group all deprecated imports not alphabatically just under one comments line and put everything there WDYT?

Comment (guoda-puidokaite) [reply packages/ui-components/src/index.js]: Nice suggestion, done.

**Review by barsukov — APPROVED** [review]
Best 🥇

File: .changeset/tasty-dragons-train.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@cloudoperators/juno-ui-components": minor
+---
+
+Marked AppIntro as deprecated in Storybook
```
Comment (guoda-puidokaite) [inline .changeset/tasty-dragons-train.md:5]: A lot of the changes here are internal. Does it make sense to have the below `changeset` message? As it's more accurate if anything went wrong...

```suggestion
Marked AppIntro as deprecated in Storybook and moved deprecated components to the deprecated_js folder in the code base (internal change)
```

Or just mention the external change?

**Review by guoda-puidokaite — COMMENTED** [review]
Question for reviewers 🙈

**Review by andypf — APPROVED** [review]
lgtm

**Pull request merged**